<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextStd</class>
 <widget class="QWidget" name="TextStd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>520</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="TextStdLeft">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="textStdRight">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextEdit" name="textStdOut">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="textStdErr">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
